Specifications/Language/2_Statements/ConditionalLoops.md

```diff
@@ -1,19 +1,20 @@
-# Conditional Loops
+# Conditional loops
 
-Much like most classical programming languages, Q# supports loops that break based on a condition, i.e. loops for which the number of iterations is unknown and may vary from run to run. Since the instruction sequence is unknown at compile time, these kinds of loops need to be handled with particular care in a quantum runtime. 
+Like most classical programming languages, Q# supports loops that break based on a condition: loops for which the number of iterations is unknown and may vary from run to run. Since the instruction sequence is unknown at compile-time, you need to handle these conditional loops with particular care in a quantum runtime. 
```

Comment:
```suggestion
Like most classical programming languages, Q# supports loops that break based on a condition: loops for which the number of iterations is unknown and may vary from run to run. Since the instruction sequence is unknown at compile-time, the compiler handles these conditional loops in a particular way in a quantum runtime. 
```

Reply:
I think what threw me off here was the phrase "handled with particular care" - that's a subjective quality for a compiler to have.